### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by mnsl**
* Fix typos in files in the modules directory (subdirectories not included)
* Rename `doc/Running And Writing Tests for Sia.md` to `doc/Running and Writing Tests for Sia.md`, which looks better and unbreaks a link at the bottom of `doc/Guide to Contributing to Sia.md`.

### CODE REVIEW

File: modules/transactionpool.go [hunk 1]
```diff
@@ -21,16 +21,16 @@ const (
...
 
 	// ErrLargeTransaction is the error that gets returned if a transaction
 	// provided to the transaction pool is larger than what is allowed by the
 	// IsStandard rules.
 	ErrLargeTransaction = errors.New("transaction is too large for this transaction pool")
 
 	// ErrLargeTransactionSet is the error that gets returned if a transaction
-	// set is given to the transaction pool if the transaction is larger than
-	// limit placed by the IsStandard rules of the transaction pool.
+	// set set given to the transaction pool is larger than the limit placed
```
Comment (lukechampine) [inline modules/transactionpool.go]: set set?

File: modules/wallet.go [hunk 2]
```diff
@@ -207,7 +207,7 @@ type (
 		// builder. Items are returned by index.
 		ViewAdded() (newParents, siacoinInputs, siafundInputs, transactionSignatures []int)
 
-		// Drop indicates that a transaction is no longer useful, will not be
+		// Drop indicates that a transaction is no longer useful and will not be
```
Comment (lukechampine) [inline modules/wallet.go]: comma seems appropriate since this is a list of 3 items

File: modules/consensus.go [hunk 3]
```diff
@@ -78,7 +78,7 @@ type (
 
 		// RevertedBlocks is the list of blocks that were reverted by the change.
 		// The reverted blocks were always all reverted before the applied blocks
-		// were applied. The revered blocks are presented in the order that they
+		// were applied. The reverted blocks are presented in the order that they
```
Comment (lukechampine) [inline modules/consensus.go]: sad to see this one go

File: modules/wallet.go (hunk 2 above)
Comment (mnsl) [inline modules/wallet.go]: the first two items have the same subject so they should not be separated by a comma, e.g. "I went to the store and picked up some apples" vs. "I went to the store, and picked up some apples"

**Comment by mnsl** [comment]
Restored revered blocks to their rightful place in consensus.go; fixed "set set" typo in transactionpool.go

File: modules/wallet.go (hunk 2 above)
Comment (lukechampine) [inline modules/wallet.go]: ah, good catch. I was interpreting the last item as speaking about the transaction directly, not its outputs.

**Pull request merged**